src/Utilities/VulnerabilityRoutes.js

```diff
@@ -0,0 +1,117 @@
+import React, { useEffect, useState, lazy, Suspense, Fragment } from 'react';
+import { Navigate, Route, Routes, useLocation } from 'react-router-dom';
+import { getSystems } from '../Helpers/APIHelper';
+import { PATHS } from '../Helpers/constants';
+import { intl } from './IntlProvider';
+import messages from '../Messages';
+import AsyncComponent from '@redhat-cloud-services/frontend-components/AsyncComponent';
+import ErrorState from '@redhat-cloud-services/frontend-components/ErrorState';
+
+const SystemsPage = lazy(() =>
+    import(
+        /* webpackChunkName: "Systems" */ '../Components/SmartComponents/SystemsPage/SystemsPage'
+    )
+);
+
+const LandingPage = lazy(() =>
+    import(
+        /* webpackChunkName: "LandingPage" */ '../Components/SmartComponents/LandingPage/LandingPage'
+    )
+);
+
+const Upgrade = lazy(() =>
+    import(
+        /* webpackChunkName: "Upgrade" */ '../Components/PresentationalComponents/StaticPages/UpgradePage'
+    )
+);
+
+const CVEDetailsPage = lazy(() =>
+    import(
+        /* webpackChunkName: "CVEDetailsPage" */ '../Components/SmartComponents/CVEDetailsPage/CVEDetailsPage'
+    )
+);
+
+const SystemDetailsPage = lazy(() =>
+    import(
+        /* webpackChunkName: "SystemDetailsPage" */ '../Components/SmartComponents/SystemDetailsPage/SystemDetailsPage'
+    )
+);
+const Reports = lazy(() =>
+    import(/* webpackChunkName: "Reports" */ '../Components/SmartComponents/Reports/ReportsPage')
+);
+
+const buildTitle = (title, location) => {
+    const subPath = location.pathname && location.pathname.split('/')[4];
+    document.title = `${subPath ? `${subPath} - ` : ''} ${title} - ${intl.formatMessage(messages.pageTitleSuffix)}`;
+};
+
+export const VulnerabilityRoutes = () => {
+    let location = useLocation();
+    const [hasSystems, setHasSystems] = useState(true);
+
+    useEffect(() => {
+        const fetchData = async () => {
+            const result = await getSystems();
+            setHasSystems(result?.meta?.total_items > 0);
+        };
+
+        fetchData();
+    }, []);
+
+    return (
+        // TODO: Discuss with UX some nice loading placeholder
+        <Suspense fallback={Fragment}>
+            {!hasSystems ? (
+                <AsyncComponent
+                    appName="dashboard"
+                    module="./AppZeroState"
+                    scope="dashboard"
+                    ErrorComponent={<ErrorState />}
+                    app="Vulnerability"
+                />
+            ) :
+                <Routes>
+                    <Route
+                        path={PATHS.cveDetailsPage.to}
+                        element={<CVEDetailsPage setTitle={() => buildTitle(intl.formatMessage(messages.cvesHeader), location)}/>}
```

Comment:
This `setTitle` function should maybe use the function from chrome to update the title, like it did before[0].

Also, it seems we don't really need to pass this function down, but can call it here in a `useEffect`, like before. 

[0] https://github.com/RedHatInsights/vulnerability-ui/pull/1905/files#diff-145599163c0e7fcdfe304a99cb21eb7e86d477c838fdbef3be47db59ec5f2875L60